docs/src/SUMMARY.md

```diff
@@ -21,6 +21,7 @@
   - [Scalars](./designing-a-schema/scalars.md)
   - [Directives](./designing-a-schema/directives.md)
   - [Relationships](./designing-a-schema/relationships.md)
+- [Generating a Schema](./generating-a-schema/index.md)
```

Comment:
This should go int the `Designing a schema` section maybe under a section called `Automatic Schema Generation` (or something that makes it clear what this section is about).